### ISSUE TITLE
--public-addr parameter not being added to systemd service file

### ISSUE THREAD

**Issue by ssssaaamm** [w3f/polkadot-validator-setup#152]
Even though I set the `enable_reverse_proxy = true` in the inventory.yml file. The template polkadot.service.j2 doesn't automatically add the `--public-addr` parameter in the polkadot.service file. And then if you enable the nginx reverse proxy you will need to add this parameter manually.

**Comment by lamafab** [comment]
It should, as seen in the [template here](https://github.com/w3f/polkadot-validator-setup/blob/master/ansible/roles/polkadot-validator/templates/polkadot.service.j2#L54):

```j2
  {% if enable_reverse_proxy is not none and enable_reverse_proxy is sameas true %}
  --public-addr=/ip4/{{ hostvars[inventory_hostname].public_ip.json.ip }}/tcp/{{ proxy_port }} \
  {% endif %}
```

Did you add that flag to the variable section of the validator? Like this:

```yaml
[validator_1:vars]
ansible_user=bob
telemetry_url=wss://telemetry.polkadot.io/submit/
logging_filter='sync=warn,afg=warn,babe=warn'
enable_reverse_proxy=true
```

**Comment by ssssaaamm** [comment]
Yes, I added the corresponding flag in the inventory.yml.

I just tried with another blank server and follow the steps from scratch just to be sure about it, and I got the same result. I configured the validator for the **polkadot** id network and **westend** chain. It creates the correct streaming Nginx configuration file and opens the 80 port and deny the p2p port as expected but in the systemd service file, it doesn't add the `--public-addr` flag. I have just switched to the polkadot chain (instead of westend) and got the same.

Here is the inventory.yml:
![image](https://user-images.githubusercontent.com/15355109/128647464-a374f075-cd9a-430e-9c5b-48e37a54c7ee.png)


And here is the systemd service file:
![image](https://user-images.githubusercontent.com/15355109/128647509-0b6c7d1f-1a6f-4812-bfd0-f14fd37be0f4.png)
